include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -56,11 +56,8 @@ namespace aspect
             set_initial_values(double para_semi_major_axis_a,
                                double para_eccentricity,
                                double para_semi_minor_axis_b,
-                               double para_rot_para_to_para_angle,
-                               double para_para_to_rect_angle,
-                               double para_rotation_longitude,
-                               double para_rotation_latitude,
-                               double para_bottom_depth);
+                               double para_bottom_depth,
+                               std::vector<std::vector<double> > para_corners);
```

Comment:
All of these are input arguments. Please mark them as `const`, and make the last argument a reference -- no need to copy a vector of points.